build.gradle

```diff
@@ -18,6 +20,15 @@ plugins {
     id 'com.github.johnrengelman.shadow' version '5.0.0'
 }
 
+def internalRepoHost = System.getenv('SNPS_INTERNAL_ARTIFACTORY')
+
+repositories {
+    maven { url "https://sig-repo.synopsys.com/bds-integrations-release" }
```

Comment:
Curious if there is a way and if it's worth it to instead re-include the `https://raw.githubusercontent.com/blackducksoftware/integration-resources/master/gradle_common/buildscript-repositories.gradle` instead of duplicating the 3 repos.

Reply:
Good point and I forgot to mention why the repositories are explicitly included. I did try this but in my case, Gradle, for an unknown reason, is unable to resolve the repositores when they are included using the `apply from..` syntax outside the buildscript closure, so for now, I decided to not pursue eliminating that redundancy as the current way does make it happy.

Reply:
@andrian-sevastyanov Build passed on Jenkins after using the `apply from` syntax. It also passed on my local Gradle after I manually stopped all Gradle daemons and restarted them. Thanks for testing this out.